lib/src/upgrade_messages.dart

```diff
@@ -46,7 +46,7 @@ enum UpgraderMessage {
 ///   String get buttonTitleIgnore => 'My Ignore';
 /// }
 ///
-/// UpgradeAlert(messages: MyUpgraderMessages());
+/// UpgradeAlert(upgrader: Upgrader(messages: MyUpgraderMessages());
```

Comment:
@amarell Good catch. However, in examples, we don't like to instantiate Upgrader() in the `build` method so it would look better like this:
```
/// final upgrader = Upgrader(messages: MyUpgraderMessages());
/// ...
/// UpgradeAlert(upgrader: upgrader);
```

Reply:
Good point. Updated.

Reply:
@amarell Thanks